### PR TITLE
Implement new button design

### PR DESCRIPTION
**Pull request by dominik-zeglen**
This PR changes the way out buttons are styled.

Screenshots:
Zrzut ekranu 2021-10-13 o 11 33 04
Zrzut ekranu 2021-10-13 o 11 33 10

### CODE REVIEW

File: src/PillLink/PillLink.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import React from "react";
+
+import useStyles from "./styles";
+
+export interface PillLinkProps
+  extends React.DetailedHTMLProps<
+    React.AnchorHTMLAttributes<HTMLAnchorElement>,
+    HTMLAnchorElement
+  > {
+  state?: "default" | "hover" | "active";
```
Comment (jwm0) [inline src/PillLink/PillLink.tsx:11]: Is there a case where we want to force active or hover state without user interaction?

File: src/PillLink/PillLink.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+export const PillLink: React.FC<PillLinkProps> = ({
+  children,
+  className,
+  state = "default",
+  ...props
```
Comment (jwm0) [inline src/PillLink/PillLink.tsx:18]: Shouldn't it have a `disabled` state?

File: src/theme/themes.ts [hunk 3]
```diff
@@ -126,6 +133,13 @@ export const light: SaleorThemeColors = {
     mid: "#FEADAD",
     light: "#FEDEDE",
   },
+  errorAction: {
+    1: "#B63755",
+    2: "#D36474",
+    3: "#E9878B",
+    4: "#F7B6B2",
+    5: "#FBDDD8",
+  },
```
Comment (orzechdev) [inline src/theme/themes.ts:142]: If some colors are the same for dark and light, maybe it is worth to consider creating `const common: SaleorThemeColors` and extending/overwriting them in `export const dark: SaleorThemeColors` and `export const light: SaleorThemeColors` with what is needed.

File: src/PillLink/PillLink.tsx (hunk 2 above)
Comment (dominik-zeglen) [reply src/PillLink/PillLink.tsx:18]: According to Pawel it doesn't

**Pull request merged**